Assets/Mirror/Editor/Weaver/Writers.cs

```diff
@@ -97,7 +105,7 @@ public static MethodReference GetWriteFunc(TypeReference variable, int recursion
             }
             else
             {
-                newWriterFunc = GenerateClassOrStructWriterFunction(variable, recursionCount);
+                newWriterFunc = GenerateClassWriterFunction(variable, recursionCount);
```

Comment:
why was it renamed?
doesn't it work for structs anymore?

Reply:
I didnt think the `OrStruct` added much value to the name, I was trying to simplify the name.

Classes and structs are treated the same when we create writer functions, and within Cecil IsClass includes both classes and structs. So i think within this context the `OrStruct` is unneeded

Reply:
@James-Frowen could we keep the old name for now? we can still discuss a name change in a separate PR, but I don't fully agree with that part yet. the rest is nice though, worth merging.